utils/visualizer/README.md

```diff
@@ -6,17 +6,31 @@ With a given qbeast table path and a target `RevisionID`, it scans the `_delta_l
 One can also provide a sampling fraction and the selected nodes will be highlighted with a different color. Details about on the sampling efficiency will be shown in the terminal. 
 
 ### Installation
-```
+
+First of all, you need to make sure you have python installed in your pc and then install poetry in order to download all the dependencies.
+
+```bash
+# We will create a virtual environment in the visualization folder
 git clone git@github.com:Qbeast-io/qbeast-spark.git
+cd your-path-to-qbeast-spark-repository/utils/visualizer
 
-virtualenv ~/.envs/index-viz
+# Pyhton version should be 3.12
+python3 --version
 
-source ~/.envs/index-viz/bin/activate
+# We will need poetry to create the environment
+pip install poetry 
 
-cd ./qbeast-spark/utils/visualizer/
+# Create a poetry shell (remember to use command exit when you are done with using this environment)

```

Comment:
I install the dependencies using `poetry install --no-root` because the --no-root is used to avoid installing the proyect itself as a package and only install the dependencies of the project. Shouldn't we keep it?

Also, I use `poetry run pip install -e .` to make sure qvis is installed. Shouldn't we also keep it or is it installed by running poetry instal?

Reply:
I would use only "poetry install"